docs/build.gradle

```diff
@@ -163,10 +168,12 @@ doxygen {
     extract_static true
     file_patterns '*'
     full_path_names true
+    full_sidebar false
     generate_html true
     generate_latex false
     generate_treeview true
     html_extra_stylesheet 'theme.css'
+    html_colorstyle 'LIGHT'
```

Comment:
Why

Reply:
So that doxygen doesn't interfere with the theme's switching. https://github.com/jothepro/doxygen-awesome-css/issues/97